client/js/utils.js

```diff
@@ -48,6 +49,15 @@ function isOpInChannel(channel) {
 	return isOP;
 }
 
+// Given a channel element will determine if the lounge user has a user mode other than norml
+function hasUserModeInChannel(channel) {
```

Comment:
We could probably find a nice way to combine these functions (allow the user to pass in a minimum privilege to check for.

Reply:
Agreed, I probably made the isOp function a it narrow in scope as I had a very specific target in mind. Maybe make it possible to pass an array of possible modes to check for so the result can just be `passedArray.contains(userMode)`

Reply:
That could be a nice way to do it.

Reply:
Alright, I've updated it to this, defaulting to all of the roles (other than normal). Let me know what you think.

Reply:
Yup looks good to me, much more versatile.